mailpoet/lib/Segments/DynamicSegments/Filters/FilterHelper.php

```diff
@@ -0,0 +1,37 @@
+<?php declare(strict_types = 1);
+
+namespace MailPoet\Segments\DynamicSegments\Filters;
+
+use MailPoet\Entities\SubscriberEntity;
+use MailPoetVendor\Doctrine\DBAL\Query\QueryBuilder;
+use MailPoetVendor\Doctrine\ORM\EntityManager;
+
+class FilterHelper {
+    /** @var EntityManager */
+  private $entityManager;
+
+  public function __construct(
+    EntityManager $entityManager
+  ) {
+    $this->entityManager = $entityManager;
+  }
+
+  public function prefixedTable(string $table): string {
```

Comment:
In accordance with our naming conventions I'd suggest something more like `getPrefixedTableName` (verb + more exact name).